modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -32,7 +37,7 @@ Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
 +
 [NOTE]
 ====
-The VolSync Operator is required only for use with the Technology Preview Data Mover. The Operator is not required for using OADP production features.
+The VolSync Operator is not required for using OADP production features.
 ====
 
 * You have installed the OADP operator by using OLM.
```

Comment:
Are there any other prerequisites?

Reply:
Volsync operator is installed, restic secret credential is created, and volumeSnapshotClass with appropriate velero label is created.

Reply:
"restic secret credential is created" - It's step 1 in the Procedure.